### PR TITLE
Silence rosdep

### LINKED ISSUE #141: Set locale in docker
Perl complains about locales..
I will fix this together with another little silence patch for rosdep.

### PR DESCRIPTION
**Pull request by ipa-mdl**
Filters most of the rosdep install output, just prints commands and errors.
closes #141

### CODE REVIEW

File: industrial_ci/src/tests/source_tests.sh [hunk 1]
```diff
@@ -117,7 +117,12 @@ fi
 
 ici_time_start rosdep_install
 
-rosdep install -q --from-paths $CATKIN_WORKSPACE --ignore-src --rosdistro $ROS_DISTRO -y
+function rosdep_install {
```
Comment (130s) [inline industrial_ci/src/tests/source_tests.sh]: What for is this a function? Are we going to reuse this somewhere?

Comment (ipa-mdl) [inline industrial_ci/src/tests/source_tests.sh]: Actually this is a little hack to make the job fail if rosdep fails.
As an alternative I could have surrounded it with
```
set -o pipefail
set +o pipefail
```

Hwoever, I plan to factor all code into functions anyway.

Comment (130s) [inline industrial_ci/src/tests/source_tests.sh]: I see. Could you then leave a short inline comment for that hack?

Comment (ipa-mdl) [inline industrial_ci/src/tests/source_tests.sh]: I have switched to pipefail and added a comment

**Pull request merged**